cli/popper/cli.py

```diff
@@ -67,3 +69,30 @@ def get_command(self, ctx, name):
 @pass_context
 def cli(ctx):
     """Popper command line interface."""
+    signal.signal(signal.SIGINT, signal_handler)
+
+
+docker_list = list()
+process_list = list()
+
+
+def signal_handler(sig, frame):
+
+    pu.info("\n")
+    cmd_out = pu.exec_cmd('docker ps -a --format "{{.Names}}"')
+    cmd_out = cmd_out[0].splitlines()
+    cmd_out = set(cmd_out)
+
+    for img in set(docker_list).intersection(cmd_out):
+        pu.exec_cmd('docker stop {}'.format(img))
+        pu.exec_cmd('docker rm -f {}'.format(img))
```

Comment:
users might have provided the `--reuse` flag to `popper run`, so we need to take that into account.

Reply:
Something else I forgot to mention is that, this also affects the message presented to the user. When `--reuse` is given, we are only stopping containers; when it isn't, we are destroying them as well.

We can also print a message like `^C signal detected, stopping running processes"

Reply:
Oh right. I will take that into account